src/app/globals.css

```
@@ -292,3 +299,160 @@ td {
+  touch-action: manipulation;
+  font-family: var(--font-boot-console), "DejaVu Sans Mono", "Liberation Mono", "Courier New", monospace;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.125;
+  font-variant-ligatures: none;
+  font-feature-settings: "liga" 0, "calt" 0;
+  font-kerning: none;
+  letter-spacing: 0;
+  -webkit-font-smoothing: none;
+  -moz-osx-font-smoothing: unset;
+  text-rendering: optimizeSpeed;
```

Stylelint (17.12.0)

[error] Expected "optimizeSpeed" to be "optimizespeed" (value-keyword-case)

(value-keyword-case)